src/main/java/com/faforever/client/game/GamesTilesContainerController.java

```diff
@@ -68,7 +67,11 @@ public GamesTilesContainerController(UiService uiService, PreferencesService pre
       JavaFxUtil.assertApplicationThread();
         while (change.next()) {
           change.getRemoved().forEach(gameInfoBean -> {
-            boolean remove = tiledFlowPane.getChildren().remove(uidToGameCard.remove(gameInfoBean.getId()));
+            Node card = uidToGameCard.remove(gameInfoBean.getId());
+            if (card != null) {
+              Tooltip.uninstall(card, tooltip);
+            }
+            boolean remove = tiledFlowPane.getChildren().remove(card);
```

Comment:
The following three lines should go into the card!=null block
The else block should also throw an error in the log.
(We don't know how many times it actually occurs and that would help I guess)